handlers/message.js

```diff
@@ -138,7 +138,7 @@ const genTypeClass = (messageType, s, proto, relNamespace) => {
             context.members.push(`/**
      * Get ${prop.name} map entry by key.
      * 
-     * @param key {String} map key
```

Comment:
I know that this is the "official" jsdoc way to describe a parameter. But this code is generated for Qooxdoo which still used the qx-style of <varname> {<vartype>}. As long as qooxdoo does not change to the official way (which is planned for version 8.0) we should stick to the qooxdoo way. I don't think that after this change qx tools like the apiviewer will still work.

Did you test the apiviewer with code generated this way? If this also works I have no problem with accepting this change.